python/tutorials/02-fused-softmax.py

```diff
@@ -137,7 +146,21 @@ def softmax(x):
         kernel._init_handles()
         n_regs = kernel.n_regs
         size_smem = kernel.metadata.shared
-        occupancy = NUM_REGS // (n_regs * WARP_SIZE * num_warps)
+        if is_hip():
+            # max_threads_per_sm returns number of regular purpose registers. On CDNA architectures this is half of all registers available.
```

Comment:
Can you double check if `max_threads_per_sm` refers number of registers? From its name it should be the max number of threads per CU.
It seems a typo, maybe you mean `NUM_REGS` here?

Reply:
Oh, yep, it is a typo. Thanks!